packages/docs/getting-started/running-a-validator-in-baklava.md

```diff
@@ -622,6 +716,196 @@ You can see additional information about your Validator, including uptime score,
 celocli validator:show $CELO_VALIDATOR_RG_ADDRESS
 ```
 
+## Registering as a Validator (Non-ReleaseGold)
+
+If you are new to validating on Baklava, then you most likely need to be following the Non-ReleaseGold steps for getting your validator up and running. ReleaseGold is done in the previous section and only concerns those who are generating accounts with ReleaseGold smart contract.
+
+### Register the Accounts
+
+You've now done all the infrastructure setup to get a validator and proxy running. The cLabs team will review your submission to receive funds and send you 12,000 testnet CELO to each of your Validator and Validator Group account addresses. These funds have no real world value but will allow you to submit transactions to the network via celocli and put up a stake to register as a validator and validator group.
+
+You can view your CELO balances by running the following commands:
+
+```bash
+# On your local machine
+celocli account:balance $CELO_VALIDATOR_GROUP_ADDRESS
+celocli account:balance $CELO_VALIDATOR_ADDRESS
+```
+
+At some point the output of these commands will change from `0` to `12e12`, indicating you have received the testnet CELO. This process involves a human, so please be patient. If you haven't received a balance within 24 hours, please get in touch again.
+
+You can also look at an account's current balance and transaction history on [Blockscout](https://baklava-blockscout.celo-testnet.org/). Enter the address into the search bar.
```

Comment:
Add: And change the net to Baklava

Reply:
Hi @barbaraliau it's linked to the Baklava blockscout. Is that ok?

Reply:
Ah, yeah I think that should be fine.